src/script/components/calling/FullscreenVideoCall.tsx

```diff
@@ -182,7 +188,19 @@ const FullscreenVideoCall: React.FC<FullscreenVideoCallProps> = ({
 
   return (
     <div id="video-calling" className="video-calling">
-      <div id="video-title" className="video-title">
+      <div
+        id="video-title"
+        className="video-title"
+        css={{height: verticalBreakpoint ? '30px' : '', lineHeight: verticalBreakpoint ? '12px' : ''}}
```

Comment:
maybe conditional object properties instead of ternaries

Reply:
Why not using css-only media definitions here? Doesn't really need to live in the js realm from what I can see

Reply:
true, that's probably the better idea since I haven't refactored anything from the less file